### DISCUSSION TITLE
migration 'reatour' to '@reactour/tour'

### DISCUSSION THREAD

**Discussion by MarianaZavarunska** [elrumordelaluz/reactour#572] [Q&A]
Hi, I am working on migration 'reatour' to **'@reactour/tour' v 3.4.0** package in my React app with SSR.  React version is **16.13.1.**  

I faced with error` (export 'TourProvider' (imported as 'TourProvider') was not found in '@reactour/tour' (possible exports: default) )`
when I wrapped the main App Component  into` TourProvider` or import `useTour `hook in my functional component.

Could you help me resolve the issue?

**Comment by elrumordelaluz** [comment]
Hi @MarianaZavarunska,
Thanks for the question! Sound an interesting opportunity to make something like a "migration guide".

> export 'TourProvider' (imported as 'TourProvider') was not found in '@reactour/tour' (possible exports: default)

Sounds weird this message if you are importing right like 

```
import { TourProvider } from '@reactour/tour'
```

Mind creating a minimal reproduction in a sandbox in order to allow others to debug your use-case and try to find a solution? Thanks!
